OAR/curl/request_doi.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
+
+import json,cgi,time
+import httplib2, sys, base64, codecs
+
+res=[]
+retCode=0
+errCode=''
+doi='11623'
+res =  "%s/sci-gaia:%s" % (doi,time.time())
+print "Content-type: application/json\n\n"
+print json.dumps(res)
```

Comment:
Yes, but it's in the cURL dir. I would expect cURL stuff in the cURL dir and python stuff in the python dir. Can you re-write this request as a cURL request, perhaps in `../../README.md` ?